### PR TITLE
add additional bars to tables

### LINKED ISSUE #786: Use percentBars/relativeBars on additional tables
Can also add them to:
* hero ranking page
* player pages (heroes/peers/pros/rankings tabs)

### PR DESCRIPTION
**Pull request by howardchung**
fixes #786 

Adds bars to more tables.

Made `percentBarsWithValue`  a separate bar type.

### CODE REVIEW

File: src/components/Player/Pages/Rankings/playerRankingsColumns.jsx [hunk 1]
```diff
@@ -10,7 +10,8 @@ export default [{
   displayName: strings.th_percentile,
   field: 'card',
   sortFn: row => row.rank / row.card,
-  displayFn: transformations.rank_percentile,
+  percentBars: row => row.rank / row.card,
```
Comment (masad-frost) [inline src/components/Player/Pages/Rankings/playerRankingsColumns.jsx]: Shouldn't this be withValue?

File: src/components/Table/Table.jsx [hunk 2]
```diff
@@ -89,9 +89,9 @@ const TableCreator = ({
               }
 
               let fieldEl = null;
-              const bars = relativeBars || percentBars;
+              const bars = relativeBars || percentBars || percentBarsWithValue;
               if (bars) {
-                const altValue = typeof bars === 'function' ? bars(row) : null;
+                const altValue = typeof bars === 'function' && percentBarsWithValue ? bars(row) : null;
```
Comment (masad-frost) [inline src/components/Table/Table.jsx]: I don't know if this is needed anymore. I'm assuming the withValue should be the only one that's a func

File: src/components/Player/Pages/Rankings/playerRankingsColumns.jsx (hunk 1 above)
Comment (howardchung) [inline src/components/Player/Pages/Rankings/playerRankingsColumns.jsx]: No, the ordinal value is shown in another column.

File: src/components/Table/Table.jsx (hunk 2 above)
Comment (howardchung) [inline src/components/Table/Table.jsx]: Can't hurt to leave it?  It enforces that percentBarsWithValue is used if we want to show an alt value

**Pull request merged**